testsuite/classlibrary/TestProxySpace.sc

```diff
@@ -0,0 +1,55 @@
+TestProxySpace : UnitTest {
+
+
+	test_storeOn_recoversState {
+		var psA, psB, stream, bValue, bNodeMap, aMapString, bMapString;
+		var server = Server(\test, NetAddr("127.0.0.1", 57111));
```

Comment:
can you please name the server `this.class.name` or something more specific to this test? makes it easier to debug and also reduces the chance of collision